reference/catalog/brands_catalog.v3.yml

```diff
@@ -523,8 +530,12 @@ paths:
         '204':
           description: ''
           content: {}
-    parameters:
-      - $ref: '#/components/parameters/Accept'
```

Comment:
Why did you remove the Accept parameter?

Reply:
Returned to code